metomi/rose/etc/tutorial/cylc-forecasting-workflow/bin/get-rainfall

```diff
@@ -23,18 +37,24 @@ from datetime import datetime
 import math
 import os
 import shutil
-import urllib.request
 
-import PIL.Image
-from mecator import get_offset, get_scale, coord_to_pos, pos_to_coord
+import requests
+
+try:
+    from PIL import Image
+except ModuleNotFoundError:
+    # not all PIL installations are created equal
+    # sometimes we must import like this
+    import Image
+from mercator import get_offset, get_scale, pos_to_coord
 import util
 
 
 URL = ('http://datapoint.metoffice.gov.uk/public/data/layer/wxobs/'
        'RADAR_UK_Composite_Highres/png?TIME={time}&key={api_key}')
 
 
-class Rainfall:
+class Rainfall(object):
```

Comment:
[Not required at python3](https://stackoverflow.com/questions/4015417/why-do-python-classes-inherit-object)

If you are intending to make this version agnostic the shebang line suggests otherwise.

Reply:
I don't think this needs to support python 2, because Rose 2 requires python 3.

But there isn't much point making changes here, better to do them on cylc-flow then rsync across